plugins/modules/zos_job_query.py

```diff
@@ -38,6 +38,8 @@
        - A job name can be up to 8 characters long.
        - The I(job_name) can contain include multiple wildcards.
        - The asterisk (`*`) wildcard will match zero or more specified characters.
+       - Note that using this value will query the system for '*' and then return just matching values
```

Comment:
Added some points for consistency.
```suggestion
       - Note that using this value will query the system for '*' and then return just matching values.
```

Reply:
Thanks for seeing that.